introspection.go

```diff
@@ -16,6 +16,7 @@ func (s *Schema) Inspect() *introspection.Schema {
 // ToJSON encodes the schema in a JSON format used by tools like Relay.
 func (s *Schema) ToJSON() ([]byte, error) {
 	result := s.exec(context.Background(), introspectionQuery, "", nil, &resolvable.Schema{
+		Meta:   s.res.Meta,
```

Comment:
In retrospect, this will work for schemas which _have_ resolvers registered, but not for schemas that have been parsed _without_ a resolver

Reply:
I've addressed this by ensuring that `s.res.Meta` is _always_ set on a parsed schema, by having `ApplyResolver` populate a resolvable schema without the entrypoints or root resolver set. This also requires that the checks before executing queries, mutations and subscriptions don't check for a `nil` resolvable schema, but rather that the resolver is missing on it